docs/src/main/sphinx/connector/base-jdbc.rst

```diff
@@ -0,0 +1,194 @@
+    connection-url=[connectorprotocol]//<host>:<port>;database=<database>
+
+    # Connection credentials
+    connection-user=USERNAME
+    connection-password=PASSWORD
+
+While the ``connector.name`` is the only required property in a catalog file,
+JDBC connectors need a specified ``connection-url``. Typically, username and
+password credentials are also required.
+
+Once the catalog properties file is established, you can add any additional
+configuration (config) properties as needed.
+
+.. _base-jdbc-config-properties:
+
+Additional config properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After establishing a base catalog properties file, you can add other
+configuration properties. Connector-specific properties are defined in the
+documentation for each connector.
+
+.. list-table:: Available config properties
+  :widths: 30, 40, 30
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default value
+  * - ``case-insensitive-name-matching``
+    - Supports case insensitive database and collection names
+    - False
+  * - ``case-insensitive-name-matching.cache-ttl``
+    -
+    - 1 minute
+  * - ``jdbc-types-mapped-to-varchar``
+    - Allow forced mapping of comma separated lists of data types to convert to
+      unbounded ``VARCHAR``
+    -
+  * - ``metadata.cache-ttl``
+    -  Duration for which meta data, including table and column statistics, is
+       cached
+    - 0 - disabled caching
+  * - ``metadata.cache-missing``
+    - Cache the fact that meta data, including table and column statistics, is
+      not available
+    - False
+
+.. _base-jdbc-performance:
+
+Performance
+-----------
+
+The JDBC connector base includes a number of features to improve connector
+performance. These features need to be extended in the connector to be available
+as user facing features.
+
+.. _base-jdbc-table-statistics:
+
+Table statistics
+^^^^^^^^^^^^^^^^
+
+Connectors supports :doc:`table and column statistics </optimizer/statistics>`.
+They are used for :doc:`cost based optimizations
+</optimizer/cost-based-optimizations>` to improve query processing performance
+based on the actual data in the data source.
+
+Collection of statistics varies for each data source.
+
+Table and column statistics are enabled by default. You can disable it for a
+catalog by setting ``statistics.enabled`` to ``false``. The ``metadata``
+properties from the preceding section further control the statistics caching.
+
+The statistics can be viewed in Trino using :doc:`/sql/show-stats`.
+
+.. _base-jdbc-join-pushdown:
+
+Join pushdown
+^^^^^^^^^^^^^
+
+Join pushdown allows the connector delegate the table join operation to the
+underlying data source. This can result in performance gains and allows Trino to
+perform the remaining query processing on a smaller amount of data.
+
+The specifics for the supported pushdown of table joins varies for each data
+source and therefore for each connector.
+
+.. _base-jdbc-topn-pushdown:
+
+TopN pushdown
```

Comment:
@martint  is "TopN" a term we would use in user-facing?

Reply:
if not.. what is the alternative?

Reply:
TopN is a familiar term to me from other jobs and other databases.

Reply:
@Ordinant can it be "topn" is a slang?

@mosabua we always refer to `ORDER BY ... LIMIT ...` in release notes.

Reply:
Google, or at least DuckDuckGo, [is familiar with](https://duckduckgo.com/?q=top+n+sql+query&t=osx&ia=web) TOP N and TOP-N. It's in [Oracle docs](https://docs.oracle.com/cd/F49540_01/DOC/server.815/a68003/rollup_c.htm). It looks like the origin is from Sybase Transact-SQL, inherited by Microsoft [in SQL Server](https://docs.microsoft.com/en-us/dax/topn-function-dax). Whether that's slang or not depends on your view of Sybase and Microsoft as standard bearers.

We'll be happy to standardize on ``ORDER BY ... LIMIT ...``, right @mosabua?

Reply:
We use ORDER BY ... LIMIT generally, as that is close to the syntax users type. There is no explicit TOP N feature in the language. That’s just an internal concept in the plan IR and connector APIs.

Reply:
I am changing this .. but the name "ORDER BY ... LIMIT pushdown" is awkward to say the least. Can we maybe at least drop the ... or so .. and just use normal case " Order by limit pushdown" ?